src/settings.js

```diff
@@ -1606,6 +1606,20 @@ var WASM2C = 0;
 // in a custom location.
 var SEPARATE_DWARF_URL = '';
 
+// Whether the program should abort when an unhandled WASM exception is encountered.
+// This makes the Emscripten program behave more like a native program where the OS
+// would terminate the process and no further code can be executed when an unhanled
+// exception (e.g. out-of-bounds memory access) happens.
+// This will instrument all exported functions to catch thrown exceptions and
+// call abort() when they happen. Once the program aborts any exported function calls
+// will fail with a "program has already aborted" exception to prevent calls into
+// code with a potentially corrupted program state.
+// This adds ~250 bytes to the code size in optimized builds and a slight overhead
```

Comment:
is it 250 even with `--closure 1`? probably less. worth mentioning just an order of magnitude figure in any case, as this may change over time
```suggestion
// This adds a small fixed amount to code size in optimized builds and a slight overhead
```